plugin/pkg/scheduler/factory/factory.go

```diff
@@ -256,8 +256,9 @@ func getNodeConditionPredicate() cache.NodeConditionPredicate {
 
 // Returns a cache.ListWatch that finds all pods that need to be
 // scheduled.
-func (factory *ConfigFactory) createUnassignedPodLW() *cache.ListWatch {
-	return cache.NewListWatchFromClient(factory.Client, "pods", api.NamespaceAll, fields.Set{client.PodHost: ""}.AsSelector())
+func (factory *ConfigFactory) createUnassignedAndRunningPodLW() *cache.ListWatch {
+	selector := parseSelectorOrDie("spec.nodeNam==" + "" + ",status.phase!=" + string(api.PodSucceeded) + ",status.phase!=" + string(api.PodFailed))
```

Comment:
typo: nodeNam==

This is probably what is causing
k8s.io/kubernetes/plugin/pkg/scheduler
test to fail. (I didn't look into it.)